kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -315,7 +316,6 @@ public void testTerminateEarlyNotExistingSolver() throws Exception {
         SolverInstance instance = new SolverInstance();
         instance.setSolverConfigFile( SOLVER_1_CONFIG );
         instance.setStatus( SolverInstance.SolverStatus.NOT_SOLVING );
-        instance.setPlanningProblem( loadPlanningProblem( 5, 15 ) );
```

Comment:
It isn't needed when terminating solver, so I removed it. For solver termination Kie server needs to know just solver status.